apps/src/code-studio/progress.js

```diff
@@ -93,6 +103,35 @@ progress.renderCourseProgress = function (scriptData, currentLevelId) {
 };
 
 /**
+ * Query the server for progress of all students in section, and render this to
+ * our teacher panel.
+ */
+// TODO - rename since this is more than teacher panel (it's lockable stuff everywhere)
```

Comment:
Hah. I could have sworn I had addressed some of these TODOs. Turns out I had, but hadn't pushed my commit :(